docs/spfx/library-component-tutorial.md

```diff
@@ -55,15 +55,24 @@ localization_priority: Priority
 1. Run `gulp` on the command prompt to see everything builds fine.
 
 ## How to consume a 3rd party SPFx library (for local testing)
-1. Run `npm link` from the root directory of library solution. In this case it would be from the **```corporate-library```** folder.
+1. Create a web part project in a **separate project folder, so not in the library project folder structure**, following the instructions from [here](https://docs.microsoft.com/en-us/sharepoint/dev/spfx/web-parts/get-started/build-a-hello-world-web-part#to-create-a-new-web-part-project). Name your web part 'CorporateWebPart'. Like the library before the web part also need to target to ***SharePoint Online only (latest)**.
  
-1. This will create a local npm link to the library with the name which is provided in the ```package.json```.
+1. From the root of the new web part folder, run the command ```npm install <path to library component>```. In this case, the command is ```npm install ..\corporate-library --save-dev```
```

Comment:
Do not use code formatting for command line... use bold formatting.

Reply:
Bad habbit my bad. Is changed now.